src/gui/folder.cpp

```diff
@@ -166,18 +165,42 @@ void Folder::checkLocalPath()
 
     if (fi.isDir() && fi.isReadable()) {
         qCDebug(lcFolder) << "Checked local path ok";
+        return true;
     } else {
         // Check directory again
         if (!FileSystem::fileExists(_definition.localPath, fi)) {
-            _syncResult.appendErrorString(tr("Local folder %1 does not exist.").arg(_definition.localPath));
-            _syncResult.setStatus(SyncResult::SetupError);
+            if (ocApp()) {
+                if (FileSystem::fileExists(fi.path() + "/..")) {
+                    int button = QMessageBox::question(ocApp()->gui()->settingsDialog(),
+                        tr("Your %1 folder %2 does not exist.").arg(ocApp()->applicationName(), _definition.localPath),
+                        tr("The folder %1 does not exist, do you want to create it or quit %2.").arg(_definition.localPath, ocApp()->applicationName()),
+                        tr("Create Folder"), tr("Quit"));
+                    if (button == 0) {
```

Comment:
```suggestion
                    if (button == QMessageBox::NoButton) {
```

Reply:
Hm not really in that case, https://code.woboq.org/qt5/qtbase/src/widgets/dialogs/qmessagebox.cpp.html#2250
I don't think switching the non obsolete question box here would be a good idea as a question that can answered with a yes or no here would be quite complex.

Reply:
Didn't see that in the docs ... hrm... .....